pyEpiabm/pyEpiabm/core/household.py

```diff
@@ -65,3 +67,32 @@ def add_person(self, person):
         """
         self.persons.append(person)
         person.household = self
+        if person.infection_status \
+                in [InfectionStatus.Susceptible]:
+            self.add_susceptible_person(person)
+
+    def add_susceptible_person(self, susceptible_person):
+        """Adds a person to the list of susceptible people in the household.
+
+        Parameters
+        ----------
+        susceptible_person : Person
+            Person to be added
+
+        """
+        if susceptible_person not in self.susceptible_persons:
+            self.susceptible_persons.append(susceptible_person)
+            susceptible_person.household = self
```

Comment:
As L43,44 append this household to the microcell's and cell's household list, do we need to update this information there?